onnxmltools/convert/coreml/operator_converters/neural_network/ImageScaler.py

```diff
@@ -4,26 +4,47 @@
 # license information.
 # --------------------------------------------------------------------------
 
+from .....proto import onnx_proto
 from ....common._registration import register_converter
+from ....common._apply_operation import apply_add, apply_mul
 
 
 def convert_preprocessing_scaler(scope, operator, container):
     params = operator.raw_operator.scaler
     # Specify some of this operator's attribute. The scale parameter in CoreML is always a scalar.
     # We just copy it and let ONNX scaler to broadcast it to all channels.
 
-    attrs = {'name': operator.full_name, 'scale': params.channelScale}
     color_space = operator.inputs[0].type.color_space
     if color_space == 'Gray8':
-        attrs['bias'] = [params.grayBias]
+        bias = [params.grayBias]
     elif color_space == 'Rgb8':
-        attrs['bias'] = [params.redBias, params.greenBias, params.blueBias]
+        bias = [params.redBias, params.greenBias, params.blueBias]
     elif color_space == 'Bgr8':
-        attrs['bias'] = [params.blueBias, params.greenBias, params.redBias]
+        bias = [params.blueBias, params.greenBias, params.redBias]
     else:
         raise ValueError('Unknown color space for tensor {}'.format(operator.inputs[0].full_name))
 
-    container.add_node('ImageScaler', [operator.inputs[0].full_name], [operator.outputs[0].full_name], **attrs)
+    if container.target_opset < 9:
```

Comment:
Did we resolve the <= decision? I though these ops only needed to be updated as part of opset 10, although it shouldn't cause any errors if they're updated as part of opset 9 since all the replacement ops exist. I was under the impression that since the experimental op still exists in opset 9, we should use that for backwards compatibility.

Reply:
This breaks runtime. @linkerzhang is fixing that by adding experimental ops back.